src/DynamoPackages/PackageLoader.cs

```diff
@@ -254,6 +254,9 @@ internal void TryLoadPackageIntoLibrary(Package package)
             // Prevent duplicate loads
             if (package.LoadState.State == PackageLoadState.StateTypes.Loaded) return;
```

Comment:
so we only load packages with load state of none?

Reply:
Well I think at the moment we allow Error states to be loaded as well
maybe we could only allow None ...
Error is pretty ambiguous at the moment (could be partially loaded in the VM...I guess)

Reply:
sounds okay to me - just curious.